.github/run_test.sh

```diff
@@ -45,6 +45,9 @@ EXCLUDE_TESTS_C8S='
 # exclude tests/tests_wireless_wpa3_owe_nm.yml and tests/tests_wireless_wpa3_sae_nm.yml
 # since failing to install mac80211_hwsim kernel module
 # exclude tests/tests_infiniband_nm.yml since missing the infiniband device
+# the `nmstate-devel` package is shipped in CRB repo, after running `dnf upgrade -y`,
+# the CRB repo will be removed and therefore the `nmstate-devel` package will not be
+# avaiable
```

Comment:
This should be possible.. I think it should be possible to add the repo. I think it is important for us to have the tests in C9S too.

Reply:
Yes, the package that contains the libnmstate is not nmstate-devel, but is python3-libnmstate.